src/puppetlabs/puppetdb/constants.clj

```diff
@@ -0,0 +1,4 @@
+(ns puppetlabs.puppetdb.constants)
+
+(def filename-forbidden-characters
```

Comment:
Agreed, that's what I was just thinking through -- we're fine as long as we don't make any non-ascii-7-bit chars "special".